arch/sim/Kconfig

```diff
@@ -98,7 +98,16 @@ config SIM_X8664_MICROSOFT
 endchoice
 
 config SIM_WALLTIME
-	bool "Execution simulation in near real-time"
+	bool "Execute the simulation in near real-time"
+	default n
+
+if SIM_WALLTIME
+choice
+	prompt "Execute the simulation in near real-time"
+	default SIM_WALLTIME_HOSTSLEEP
+
+config SIM_WALLTIME_HOSTSLEEP
```

Comment:
change to SIM_WALLTIME_SLEEP like SIM_WALLTIME_SIGNAL

Reply:
Ok, I will update it